### PR TITLE
docs: Add v2022.1.4 release notes

### PR DESCRIPTION
**Pull request by blocktrron**
Add Release notes for the Gluon Release v2022.1.4.

The commit adding Release-Notes requires to be forward-ported to master.

### CODE REVIEW

File: docs/releases/v2022.1.4.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  
+- Mercusys
+
+  - MR70X (v1)
+
+- Xiaomi
+
+  - Mi Router 3G
+  
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:54]:
```suggestion


```

lines contain two spaces each

File: docs/releases/v2022.1.4.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  - RUT230 (v1)
+
+
+ath79-nand
+~~~~~~~~~~
+
+- NETGEAR
+
+  - WNDR4300 (v1)
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:25]:
```suggestion

```

line contains two spaces. I'm not arguing to remove the line, but the spaces. GitHubs UI is weird regarding empty lines.

File: docs/releases/v2022.1.4.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  - Mi Router 3G
+  
+  
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - RE200 (v3)
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:61]:
```suggestion

```

line contains two spaces

File: docs/releases/v2022.1.4.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  - RE200 (v3)
+  
+
+realtek-rtl838x
+~~~~~~~~~~~~~~~
+
+- D-Link
+
+  - DGS-1210-10P
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:69]:
```suggestion

```

line contains two spaces

File: docs/releases/v2022.1.4.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+- AVM
+
+  - FRITZBox 7520
+  
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:78]:
```suggestion


```

lines contain two spaces each

File: docs/releases/v2022.1.4.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+--------
+
+ * Enterasys WS-AP3705i now uses the correct image-name for use with the autoupdater
+   (`#2819 <https://github.com/freifunk-gluon/gluon/issues/2819>`_)
+
+ * Reduce memory Usage for ath10k on ZyXEL WRE6606 devices
+   (`#2842 <https://github.com/freifunk-gluon/gluon/issues/2842>`_)
+
+ * Replace the Workaround for failed boots on ath79 with a proper fix.
+   (`#2784 <https://github.com/freifunk-gluon/gluon/issues/2784#issuecomment-1452126501>`_)
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:97]:
```suggestion
* Enterasys WS-AP3705i now uses the correct image-name for use with the autoupdater
  (`#2819 <https://github.com/freifunk-gluon/gluon/issues/2819>`_)

* Reduce memory Usage for ath10k on ZyXEL WRE6606 devices
  (`#2842 <https://github.com/freifunk-gluon/gluon/issues/2842>`_)

* Replace the Workaround for failed boots on ath79 with a proper fix
  (`#2784 <https://github.com/freifunk-gluon/gluon/issues/2784#issuecomment-1452126501>`_)
```

Amount of left padding spaces should be a multiple of two, like in the "Known issues" section.

**Review by AiyionPrime — CHANGES_REQUESTED** [review]
Mainly linting stuff.

File: docs/releases/v2022.1.4.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+  - LibreRouter (v1)
+
+- Teltonika
+
+  - RUT230 (v1)
+
+
+ath79-nand
+~~~~~~~~~~
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:21]:
```suggestion

- Aerohive

  - HiveAP 121
  
```

File: docs/releases/v2022.1.4.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+- Arcadyan
+
+  - o2 Box 6431
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- D-Link
```
Comment (blocktrron) [inline docs/releases/v2022.1.4.rst:38]: cudy x6 v1

Comment (AiyionPrime) [reply docs/releases/v2022.1.4.rst:38]: cudy x6 v2 #2891

File: docs/releases/v2022.1.4.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+~~~~~~~~~~~~~
+
+- Arcadyan
+
+  - o2 Box 6431
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
```
Comment (AiyionPrime) [inline docs/releases/v2022.1.4.rst:37]:
```suggestion

- Cudy

  - X6 (v1, v2)

```

**Review by AiyionPrime — CHANGES_REQUESTED** [review]

File: docs/releases/v2022.1.4.rst (hunk 8 above)
Comment (AiyionPrime) [reply docs/releases/v2022.1.4.rst:38]: https://github.com/freifunk-gluon/gluon/pull/2878#discussion_r1186516291

**Comment by blocktrron** [comment]
I think we can continue here. In case somebody reall REALLY wants to have #2879 here, I expect the corresponding reference for the Release-Notes as a deliverable.

**Comment by AiyionPrime** [comment]
I really want that in the release and think so does your community.
There are several ath79 devices that are effectively unusable in the current state on stable and @herbetom has not indicated negative experiences with this backport, yet.

File: docs/releases/v2022.1.4.rst [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+ * Enterasys WS-AP3705i now uses the correct image-name for use with the autoupdater
+   (`#2819 <https://github.com/freifunk-gluon/gluon/issues/2819>`_)
+
+ * Reduce memory Usage for ath10k on ZyXEL WRE6606 devices
+   (`#2842 <https://github.com/freifunk-gluon/gluon/issues/2842>`_)
+
+ * Replace the Workaround for failed boots on ath79 with a proper fix.
+   (`#2784 <https://github.com/freifunk-gluon/gluon/issues/2784#issuecomment-1452126501>`_)
+
```
[the remark this answers is not in the document]
Comment (AiyionPrime) [reply docs/releases/v2022.1.4.rst:106]: This is the deliverable you requested, you just chose not to apply it yet, I think.

**Review by AiyionPrime — CHANGES_REQUESTED** [review]

**Review by AiyionPrime — CHANGES_REQUESTED** [review]
As requested: the 'resolved' changes from a few weeks ago in patch form:
https://aiyionpri.me/shared/blocktrron/0001-fixup-docs-Add-v2022.1.4-release-notes.patch

File: docs/releases/v2022.1.4.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (AiyionPrime) [reply docs/releases/v2022.1.4.rst:106]: applied in line 109

**Review by AiyionPrime — APPROVED** [review]
Looks good; I'd suggest putting [docs/releases/v2022.1.4.rst](https://github.com/freifunk-gluon/gluon/pull/2878/files#diff-11b2ede48a1e00e492a520982a7bcbcb02d2166c089ce69f16557a72bd48a5f4) onto master and run the the editorconfig checker, before that thing bugs you upon forwardporting it later.
Currently at work.

**Pull request merged**